src/Runner.js

```diff
@@ -90,11 +93,10 @@ function run(transformFile, paths, options) {
     return;
   }
 
-  getAllFiles(paths)
-    .then(files => files.filter(
-      name => !extensions || extensions.indexOf(path.extname(name)) != -1
-    ))
-    .then(files => {
+  getAllFiles(
+    paths,
+    name => !extensions || extensions.indexOf(path.extname(name)) != -1
```

Comment:
Thinking about it, this is kind of weird IMO. `js` would also match `jsx` and `json`. I think this should be stricter.